src/views/Correction.vue

```diff
@@ -28,13 +28,14 @@
 
     <CompletingParty class="mt-10" sectionNumber="1." />
 
-    <Detail class="mt-10" sectionNumber="2." />
+    <Detail class="mt-10" sectionNumber="2." validate="true" />
 
-    <CertifySection class="mt-10" sectionNumber="3." />
+    <CertifySection class="mt-10" sectionNumber="3." validate="true" />
 
     <StaffPayment
       class="mt-10"
       sectionNumber="4."
+      validate="true"
```

Comment:
These are to show the red styling right away in a correction filing (since there is otherwise no indicator what section is invalid -- you only see that the File and Pay button is disabled).

This will be refactored soon anyway, as correction filings will become 2 pages, same as the other filings in this UI.